packages/axios-extension/src/abap/abap-service-provider.ts

```diff
@@ -130,16 +130,15 @@ export class AbapServiceProvider extends ServiceProvider {
     /**
      * Create or get an existing instance of the UI5 ABAP repository service.
      *
+     * @param alias - optional alias path on which the UI5Repository service is exposed
      * @returns an instance of the UI5 ABAP repository service.
      */
-    public getUi5AbapRepository(): Ui5AbapRepositoryService {
-        if (!this.services[Ui5AbapRepositoryService.PATH]) {
-            this.services[Ui5AbapRepositoryService.PATH] = this.createService<Ui5AbapRepositoryService>(
-                Ui5AbapRepositoryService.PATH,
-                Ui5AbapRepositoryService
-            );
+    public getUi5AbapRepository(alias?: string): Ui5AbapRepositoryService {
+        const path = alias ?? Ui5AbapRepositoryService.PATH;
```

Comment:
Are we ok with `''` as an input for alias?

Reply:
I have to admit, I hadn't thought about it but, yes, we are ok with it, we should/could even use it for BAS scenarios, where a destination points to a full service url and not just a system.